hazelcast-hibernate52/src/main/java/com/hazelcast/hibernate/local/CleanupService.java

```diff
@@ -31,19 +33,23 @@
     private final Duration fixedDelay;
     private final String name;
     private final ScheduledExecutorService executor;
+    private final List<Runnable> cacheDestroy;
 
     public CleanupService(final String name, final Duration fixedDelay) {
         this.fixedDelay = fixedDelay;
         this.name = name;
         executor = Executors.newSingleThreadScheduledExecutor(new CleanupThreadFactory());
+        cacheDestroy = new LinkedList<>();
```

Comment:
Any particular reason why `LinkedList` is better over `ArrayList`?

Reply:
Since is was just add at the end, one iteration on stop and an unknown size, I thought it would perform better. But doing some review, the cases an ArrayList would underperform are too edgy, so we should go with an ArrayList. I'll move to that.